cmd/kusk/Makefile

```diff
@@ -64,7 +64,7 @@ install-deps:
 .PHONY: build
 build: install-deps
 	cd ../../config/default && kustomize edit set image kusk-gateway=${MANAGER_IMG}
-	go generate
+	go generate ./...
```

Comment:
three dots? does this work?

Reply:
Yes.

Reply:
just tested it and it works

Reply:
@aabedraba `./...` is a recursive operator to go into every directory and traverse every .go file